toucan_connectors/google_big_query/google_big_query_connector.py

```diff
@@ -84,3 +113,36 @@ def _retrieve_data(self, data_source: GoogleBigQueryDataSource) -> pd.DataFrame:
             },
         )
         return result
+
+    @staticmethod
+    def _prepare_parameters(query: str, parameters: Dict) -> List:
+        query_parameters = []
+        # replace ToucanToco variable definition by Google Big Query variable definition
+        for k in parameters:
+            if query.find('@' + k) > -1:
+                # set all parameters with a type defined and necessary for Big Query
+                query_parameters.append(
+                    bigquery.ScalarQueryParameter(k, _define_type(parameters[k]), parameters[k])
+                )
+        return query_parameters
+
+    @staticmethod
+    def _prepare_query(query: str) -> str:
+        # replace ToucanToco variable definition by Google Big Query variable definition
+        new_query = query.replace('{{', '@').replace('}}', '')
+        return new_query
```

Comment:
because it's just a replacement... is it possible to just return the final result instead of creating a variable for only one instruction 🤔 ? like : 
```python
return query.replace('{{', '@').replace('}}', '')
```

in a big overwiew... if we are sure this is the only operation we're doing... do we need a whole function to store that operation ? or maybe am wrong... and others steps are coming ? 🤔